code/game/machinery/vending_types.dm

```diff
@@ -359,9 +362,10 @@
 		/obj/item/weapon/grenade/flashbang = 4
 	)
 	restock_blocked_items = list(
-		/obj/item/weapon/storage/box/donut, 
+		/obj/item/weapon/storage/box/donut,
```

Comment:
Just readd the donut boxes to the security vendor.
It seems I just fogot to add them to the products list and only added them to the restock_blocked_items

Reply:
donuts boxes are still there in the premium list as before (i presume that's what coins get you)